pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -37,10 +36,6 @@ const (
 	AlertmanagerConfigKindKey = "alertmanagerconfig"
 )
 
-var (
-	opsGenieTypeRe = regexp.MustCompile("^(team|user|escalation|schedule)$")
```

Comment:
The regexp isn't used anywhere so I assume that it got lost in some refactoring a while ago but the validation should apply to the OpsGenieConfigResponder's type.

Reply:
I don't think it should since from https://prometheus.io/docs/alerting/latest/configuration/#responder it is a template string

```yaml
# "team", "user", "escalation" or schedule".
type: <tmpl_string>
```

Reply:
ah my bad!